modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/kb_article_tools.jsp

```diff
@@ -33,207 +31,15 @@ int status = (Integer)request.getAttribute(KBWebKeys.KNOWLEDGE_BASE_STATUS);
-				method="get"
-				url='<%=
-					PortletURLBuilder.create(
-						PortalUtil.getControlPanelPortletURL(renderRequest, KBPortletKeys.KNOWLEDGE_BASE_ADMIN, PortletRequest.RENDER_PHASE)
-					).setMVCPath(
-						"/admin/common/edit_kb_article.jsp"
-					).setRedirect(
-						redirect
-					).setParameter(
-						"parentResourceClassNameId", kbArticle.getClassNameId()
-					).setParameter(
-						"parentResourcePrimKey", kbArticle.getResourcePrimKey()
-					).buildString()
-				%>'
-			/>
-		</c:if>
-
-		<c:if test="<%= kbArticle.isRoot() && KBArticlePermission.contains(permissionChecker, kbArticle, KBActionKeys.PERMISSIONS) %>">
-			<liferay-security:permissionsURL
-				modelResource="<%= KBArticle.class.getName() %>"
-				modelResourceDescription="<%= kbArticle.getTitle() %>"
-				resourcePrimKey="<%= String.valueOf(kbArticle.getResourcePrimKey()) %>"
-				var="permissionsURL"
-				windowState="<%= LiferayWindowState.POP_UP.toString() %>"
-			/>
-
-			<liferay-ui:icon
-				icon="lock"
-				label="<%= true %>"
-				markupView="lexicon"
-				message="permissions"
-				method="get"
-				url="<%= permissionsURL %>"
-				useDialog="<%= true %>"
-			/>
-		</c:if>
-
-		<c:if test="<%= KBArticlePermission.contains(permissionChecker, kbArticle, KBActionKeys.MOVE_KB_ARTICLE) %>">
-			<liferay-ui:icon
-				icon="move"
-				label="<%= true %>"
-				markupView="lexicon"
-				message="move"
-				method="get"
-				url='<%=
-					PortletURLBuilder.create(
-						PortalUtil.getControlPanelPortletURL(renderRequest, KBPortletKeys.KNOWLEDGE_BASE_ADMIN, PortletRequest.RENDER_PHASE)
-					).setMVCPath(
-						"/admin/common/move_object.jsp"
-					).setRedirect(
-						redirect
-					).setParameter(
-						"parentResourceClassNameId", kbArticle.getParentResourceClassNameId()
-					).setParameter(
-						"parentResourcePrimKey", kbArticle.getParentResourcePrimKey()
-					).setParameter(
-						"resourceClassNameId", kbArticle.getClassNameId()
-					).setParameter(
-						"resourcePrimKey", kbArticle.getResourcePrimKey()
-					).buildString()
-				%>'
-			/>
-		</c:if>
-
-		<c:if test="<%= KBArticlePermission.contains(permissionChecker, kbArticle, KBActionKeys.DELETE) %>">
-			<liferay-portlet:renderURL var="homeURL">
-				<portlet:param name="mvcRenderCommandName" value="/knowledge_base/view" />
-			</liferay-portlet:renderURL>
-
-			<liferay-portlet:actionURL name="/knowledge_base/delete_kb_article" var="deleteURL">
-				<portlet:param name="redirect" value="<%= homeURL %>" />
-				<portlet:param name="resourceClassNameId" value="<%= String.valueOf(kbArticle.getClassNameId()) %>" />
-				<portlet:param name="resourcePrimKey" value="<%= String.valueOf(kbArticle.getResourcePrimKey()) %>" />
-				<portlet:param name="status" value="<%= String.valueOf(status) %>" />
-			</liferay-portlet:actionURL>
-
-			<liferay-ui:icon-delete
-				label="<%= true %>"
-				showIcon="<%= true %>"
-				url="<%= deleteURL %>"
-			/>
-		</c:if>
+		<clay:dropdown-actions
```

Comment:
Hey @marcogalluzzi,
There is an accessibility issue here, you should add an aria label here like we have in other places of portal, it should be something like:

```
<clay:dropdown-actions
	aria-label='<%= LanguageUtil.get(request, "show-actions") %>'
	...
/>
```

cc: @p2kmgcl, Another accessibility issue here, 😉 . Do we have the sf rule?

Reply:
Thanks @ealonso! I'll add this `aria-label` in a next pull. We are waiting to this pull to get merged to complete other tasks, and this is huge enough, so I would prefer not to stop this PR because of this reason. For sure I'll add it!

Reply:
> cc: @p2kmgcl, Another accessibility issue here, wink . Do we have the sf rule?

Not yet, but I am requesting that right know. Thanks for the warning!